pennylane/devices/default_qubit.py

```diff
@@ -599,7 +599,7 @@ def density_matrix(self, wires):
 
         # Return the full density matrix by using numpy tensor product
         if wires == self.wires:
-            density_matrix = self._tensordot(state, self._conj(state), 0)
+            density_matrix = self._tensordot(state, self._conj(state), axes=0)
```

Comment:
Wait, was this a bug?

Reply:
Ah, no. This I did for clarity, but actually intermediately. Will revert.